packages/teams-js/src/internal/handlers.ts

```diff
@@ -44,6 +44,9 @@ export function callHandler(name: string, args?: unknown[]): [true, unknown] | [
     callHandlerLogger('Invoking the registered handler for message %s with arguments %o', name, args);
     const result = handler.apply(this, args);
     return [true, result];
+  } else if (Communication.childWindow) {
+    sendMessageEventToChild(name, [args]);
+    return [true, undefined];
```

Comment:
I think this should be false instead of true, since the boolean signifies whether or not the handler was called and in this case it wasn't (it was proxied to the child window in case the child window has a handler for it)

Reply:
That seems reasonable.  I wasn't really sure what I should return, I will change it to false.